### ISSUE TITLE
plugin slows down Glyphs

### ISSUE THREAD

**Issue by laerm0** [Mark2Mark/Show-Node-Count#1]
Hi, I've had this warning for several months now.

![screenshot_2019-06-12_10-44-36](https://user-images.githubusercontent.com/10780161/59360995-22ce3800-8cff-11e9-9261-ad825b59f7f3.png)

I can't say I've noticed a performance degradation, but I'm not an expert.

**Comment by schriftgestalt** [comment]
Do you have the latest version of the plugin. Because I don’t get that error message.

**Comment by laerm0** [comment]
Yes: checking for updates in `Preferences -> Addons -> Plugins` does not show any updates for this. I am on Glyphs 2.6.1 (1230).

**Comment by laerm0** [comment]
I will try to uninstall and reinstall.

**Comment by laerm0** [comment]
Huh: this doesn't show up when I search for "node" in the plugin manager. 🤷🏼‍♀️

**Comment by laerm0** [comment]
<bump> Any reason why this disappeared from Glyphs' plugin manager?

**Comment by Mark2Mark** [comment]
> Any reason why this disappeared from Glyphs' plugin manager?

Erm, dunno? @schriftgestalt @mekkablue

**Comment by laerm0** [comment]
Pinging you @schriftgestalt and @mekkablue to check and see why this has disappeared from Glyphs' plugin manager. Thanks!

**Comment by schriftgestalt** [comment]
Has it ever beed in the Plugin Manager? I went through some of the history and couldn't find it.

**Comment by laerm0** [comment]
Oh, perhaps not. Time to add it? :)
